src/containers/AccountModal/AccountModal.tsx

```diff
@@ -60,6 +61,22 @@ const AccountModal = () => {
     }
   }, [viewParam, loading, isPublicView, user, toLogin]);
 
+  // handle websocket notifications
+  useEffect(() => {
+    if (user) {
+      const handleNotification = async () => {
+        if (user) {
+          await subscribeToNotifications(user.uuid, async (navigateTo) => {
```

Comment:
Do we automatically close socket connection on the logout? My idea was to do it when the user is not defined anymore.

Reply:
Yes, we do it along with the logout

Reply:
I don't think it's possible though.. If the user is not defined, we cannot call anything from the InPlayer sdk